src/devices/amina.ts

```diff
@@ -0,0 +1,357 @@
+                access: 'STATE',
+            }),
+
+            binary({
+                name: 'alarm_active',
+                cluster: 'aminaControlCluster',
+                attribute: 'alarmActive',
+                description: 'An active alarm is present',
+                valueOn: ['True', 1],
+                valueOff: ['False', 0],
+                access: 'STATE',
+            }),
+
+            electricityMeter({
+                cluster: 'electrical',
+                acFrequency: true,
+                threePhase: true,
+            }),
+
+            binary({
+                name: 'single_phase',
+                cluster: 'aminaControlCluster',
+                attribute: 'singlePhase',
+                description: 'Enable single phase charging. A restart of charging is required for the change to take effect.',
+                valueOn: ['Enable', 1],
+                valueOff: ['Disable', 0],
+                entityCategory: 'config',
+            }),
+
+            binary({
+                name: 'enable_offline',
+                cluster: 'aminaControlCluster',
+                attribute: 'enableOffline',
+                description: 'Enable offline mode when connection to the network is lost',
+                valueOn: ['Enable', 1],
+                valueOff: ['Disable', 0],
+                entityCategory: 'config',
+            }),
+
+            numeric({
+                name: 'time_to_offline',
+                cluster: 'aminaControlCluster',
+                attribute: 'timeToOffline',
+                description: 'Time until charger will behave as offline after connection has been lost',
+                valueMin: 0,
+                valueMax: 60,
+                valueStep: 1,
+                unit: 's',
+                entityCategory: 'config',
+            }),
+
+            numeric({
+                name: 'offline_current',
+                cluster: 'aminaControlCluster',
+                attribute: 'offlineCurrent',
+                description: 'Maximum allowed amperage draw when device is offline',
+                valueMin: 6,
+                valueMax: 32,
+                valueStep: 1,
+                unit: 'A',
+                entityCategory: 'config',
+            }),
+
+            binary({
+                name: 'offline_single_phase',
+                cluster: 'aminaControlCluster',
+                attribute: 'offlineSinglePhase',
+                description: 'Use single phase charging when device is offline',
+                valueOn: ['Enable', 1],
+                valueOff: ['Disable', 0],
+                entityCategory: 'config',
+            }),
+        ],
+
+        endpoint: (device) => {
+            return {default: 10};
+        },
+
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(10);
+
+            const binds = ['genBasic', 'genOnOff', 'haElectricalMeasurement', 'genLevelCtrl', 'aminaControlCluster'];
```

Comment:
```suggestion
            const binds = ['genBasic', 'genLevelCtrl', 'aminaControlCluster'];
```

Done by `onOff`/`electricityMeter`

Reply:
Fixed in https://github.com/Koenkk/zigbee-herdsman-converters/pull/8191/commits/99fdbdd79895b94f98a2df89c24d8c3b0bbb4b68